### ISSUE TITLE
FileManager: File opening/highlighting should not require two arguments

### ISSUE THREAD

**Issue by linusg** [SerenityOS/serenity#5597]
Currently, to open `ReadMe.md` in FileManager via the Terminal / spawing the process via LibDesktop requires this:

```
FileManager /home/anon Readme.md
```

It should be smart enough to figure this out on its own (should have the same outcome as the above):

```
FileManager /home/anon/Readme.md
```

which translates to this short form :)

```
fm R<tab><enter>
```

**Comment by linusg** [comment]
Nautilus for example has

```
-s, --select               Select specified URI in parent folder.
```

i.e. `nautilus file` and `nautilus file -s` both open `file`'s directory and select it, `nautilus /foo/bar` opens (the directory) `/foo/bar` and `nautilus /foo/bar -s` opens `/foo` and selects `bar`.

**Comment by linusg** [comment]
Implemented in #5595.
